### DISCUSSION TITLE
Help with vercel / nextjs-subscription-payments repo?

### DISCUSSION THREAD

**Discussion by silvanet** [vercel/community#863] [Help]
I cloned this repo and set it up in a Supabase Docker container. I can get it up and running but am having a little trouble understanding the Stripe setup. I followed the instructions in the repo. I have set up very basic Stripe apps before. I deployed it to Vercel and also have a local testing and development version. I set up the variables on Vercel. My Supabase database is working. When I log in a user, a pricing list should show, but it doesn't. In the Stripe webhook listening section I see errors. I note that the webhook there is not the same as my localhost webhook. Should I not be listening on localhost from the CLI at all? I'm confused.

**Comment by amyegan** [comment]
Hi @silvanet. Can you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) or your test repo so I can get a better understanding of the problem you're facing?

**Reply by silvanet** [reply]
Sorry, did I do that? I had also sent a question for help to support.

**Reply by amyegan** [reply]
I don't quite understand what you are asking here. Can you please provide a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) and step-by-step description of what you are doing? That will help me see what is going wrong and help you find a way to fix it

**Comment by silvanet** [comment]
Yes, I've kept working on it and ironed out some of my problems, but please
do take a look at it and if you can give me some guidance or suggestions
I'd greatly appreciate it. I have deployed it to Vercel. I don't think I
paid close attention when I was creating the environment variables. I don't
think all of them should have had a production environment. I'd like to
continue working on it, testing it on my localhost. I made my github repo a
private one for the moment. It has been a big project for me to get all the
little details working. It's at
https://supabase-sample-jw8rdh1n0-silvanet-gmailcom.vercel.app.

I got the basic pricing table showing up now and all the data is being sent
to my Supabase database, although the users' full name, avatar url, and
billing address aren't coming across. Maybe that's because I'm in Stripe's
testing sandbox. I have to do a whole lot more deep diving into the code to
understand it.



I love you guys!
